spring-cloud-dataflow-docs/src/main/asciidoc/howto.adoc

```diff
@@ -18,33 +17,8 @@ can send us a {github-code}[pull request].
 
 == Configure Remote Maven Repository
 
-If you'd like to connect to your organization's internal maven repository, you can override the default
-repository location by passing the credentials via the following environment properties:
-
-[source,bash]
-----
-export MAVEN_LOCAL_REPOSITORY=mylocalMavenRepo
-export MAVEN_REMOTE_REPOSITORIES_REPO1_URL=https://repo1
-export MAVEN_REMOTE_REPOSITORIES_REPO1_AUTH_USERNAME=user1
-export MAVEN_REMOTE_REPOSITORIES_REPO1_AUTH_PASSWORD=pass1
-export MAVEN_REMOTE_REPOSITORIES_REPO2_URL=https://repo2
-----
-
-If you are running behind a proxy, then you need to specify the proxy settings via environment properties
-like this:
-
-[source,bash]
-----
-export MAVEN_PROXY_PROTOCOL=https
-export MAVEN_PROXY_HOST=host1
-export MAVEN_PROXY_PORT=8090
-export MAVEN_PROXY_NON_PROXY_HOSTS='host2|host3'
-export MAVEN_PROXY_AUTH_USERNAME=user1
-export MAVEN_PROXY_AUTH_PASSWORD=passwd
-----
-
-Alternatively, you can pass them wrapped inside `SPRING_APPLICATION_JSON` environment property like
-the following:
+If you'd like to connect to a "custom" remote maven repository, you can override the default
```

Comment:
While we specify the environment properties way of configuring maven properties, we also need to specify the same can be achieved via command line properties as the same is applicable across server implementations.